### PR TITLE
Copy options and fuzzing dictionary to $OUT.

### PR DESCRIPTION
**Pull request by PhilipHazel**
This is my first attempt at editing build.sh. Please check carefully that I haven't screwed up.

### CODE REVIEW

**Comment by PhilipHazel** [comment]
I signed it!

File: projects/pcre2/build.sh [hunk 1]
```diff
@@ -21,6 +21,8 @@
 make -j$(nproc) clean
 make -j$(nproc) all
 
+cp $SRC/pcre2_fuzzer.dict $SRC/pcre2_fuzzer.options $OUT/
```
Comment (Dor1s) [inline projects/pcre2/build.sh]: I don't see .options file in the pcre2 repo: https://github.com/svn2github/pcre2

Comment (Dor1s) [inline projects/pcre2/build.sh]: Could you also move this line to the end of the script to be executed after compilation of the fuzzer.

Comment (PhilipHazel) [inline projects/pcre2/build.sh]: I moved the line to the end of the script, and seem to have created a new pull request. The .options file was only committed to the PCRE2 SVN repo today, so perhaps the github repo hasn't pulled it yet.

**Pull request closed without merging**